### PR TITLE
Adjusts Meteor Required Players

### PR DESCRIPTION
**Pull request by Fox-McCloud**
It's literally impossible for a small crew to deal with meteors, nor is it fun for them; it's just getting pelted endlessly until the crew inevitably calls the shuttle.

During higher population rounds construction/rebuilding can be engaged it, but lower crew rounds just doesn't afford this luxury.

Required players: 35

### CODE REVIEW

**Comment by Plotron** [comment]
Damn it, next time put the number in your post, so I won't have to click over to Files Changed.

**Comment by Tastyfish** [comment]
I would argue that a 35 person crew probably means 1 or 2 people in engineering at most, and thus is still too low, but better than 0.

**Comment by Fox-McCloud** [comment]
@Tastyfish 

Any suggestions on numbers?

**Comment by Tastyfish** [comment]
Honestly not sure, although I've personally never been clear on how long it's expected to go, if we're supposed to hold out until the bridge is literally being slammed or what. I've been on an engineering team with 5 other engineering crew and we were unable to keep up after 2 waves beyond minimally keeping the main areas pressurized. (Security was literally sliced in half, and medbay was almost in half, though.) I guess the crew count depends on how long an average meteor round is expected to go. 
*TLDR:* I have no idea, 35 is probably fine.

**Comment by Robertha89** [comment]
Uh, using how many engineers that should be in a round of that many people is bad, because often that number is 0. Now, what you do is make a var that counts how many engineers it is, if it's less then 0, let it be extended, and only let the CE and Station Engineers be counted, it's like how the previous event manager worked. Anyway, the more engineers, the more meteors, because everyone knows that all of those meteors got an NT stamp on them.

**Comment by Earthdivine** [comment]
To be honest, 35 people, even without engineers, is fine. There's nothing to keep the HoP from throwing Engineering access to people, as his office is FAR away from the Meteors, Borgs can do module switches, and people more then likely will hack their way in if the AI isn't already letting people in.
